.travis.yml

```diff
@@ -0,0 +1,5 @@
+language: node_js
+node_js:
+  - "5.10.1"
```

Comment:
I couldn't get npm to install gulp properly with node version `4.4.3` (I'm guessing it has something to do with an older version of npm).

This shouldn't mater because we don't have any server side tests yet (and the tests are running in PhantomJS any way).

@joemccann I'l fix this issue when setup server side tests (which I want to discuss with you on Slack).